src/Application/GameLuaBindings.cpp

```diff
@@ -0,0 +1,317 @@
+    }
+    value = &pParty->pCharacters[characterIndex - 1];
+    return 0;
+}
+
+static int lua_party_get(lua_State *L) {
+    lua_check_ret(lua_check_args(L, lua_gettop(L) == 2));
+
+    const char *name = luaL_checkstring(L, 1);
+    int value;
+
+    if (!strcmp(name, "food"))
+        value = pParty->GetFood();
+    else if (!strcmp(name, "gold"))
+        value = pParty->GetGold();
+    else
+        return luaL_argerror(L, 2, lua_pushfstring(L, "name '%s' is unknown", name));
+
+    lua_pushinteger(L, value);
+
+    return 1;
+}
+
+static int lua_party_give(lua_State *L) {
+    lua_check_ret(lua_check_args(L, lua_gettop(L) == 3));
+
+    const char *name = luaL_checkstring(L, 1);
+    int value = luaL_checknumber(L, 2);
+
+    if (!strcmp(name, "food"))
+        pParty->GiveFood(value);
+    else if (!strcmp(name, "gold"))
+        pParty->AddGold(value);
+    else
+        return luaL_argerror(L, 2, lua_pushfstring(L, "name '%s' is unknown", name));
+
+    return 0;
+}
+
+static int lua_party_set(lua_State *L) {
+    lua_check_ret(lua_check_args(L, lua_gettop(L) == 3));
+
+    const char *name = luaL_checkstring(L, 1);
+    int value = luaL_checknumber(L, 2);
+
+    if (!strcmp(name, "food"))
+        pParty->SetFood(value);
+    else if (!strcmp(name, "gold"))
+        pParty->SetGold(value);
+    else
+        return luaL_argerror(L, 2, lua_pushfstring(L, "name '%s' is unknown", name));
+
+    return 0;
+}
+
+static int lua_party_take(lua_State *L) {
+    lua_check_ret(lua_check_args(L, lua_gettop(L) == 3));
+
+    const char *name = luaL_checkstring(L, 1);
+    int value = luaL_checknumber(L, 2);
+
+    if (!strcmp(name, "food"))
+        pParty->TakeFood(value);
+    else if (!strcmp(name, "gold"))
+        pParty->TakeGold(value);
+    else
+        return luaL_argerror(L, 2, lua_pushfstring(L, "name '%s' is unknown", name));
+
+    return 0;
+}
+
+static int lua_party_member_get(lua_State *L) {
+    lua_check_ret(lua_check_args(L, lua_gettop(L) == 3));
+
+    int playerID = luaL_checkinteger(L, 1);
+    const char *name = luaL_checkstring(L, 2);
+    int actual = 0, base = 0, modifier = 0;
+
+    if (playerID < 1 || playerID > 4)
+        return luaL_argerror(L, 2, lua_pushfstring(L, "member id '%d' is invalid", playerID));
+
+    Character player = pParty->pCharacters[playerID - 1];
+    if (!strcmp(name, "age")) {
+        actual = player.GetActualAge();
+        base = player.GetBaseAge();
+        modifier = player.sAgeModifier;
+    } else if (!strcmp(name, "accuracy")) {
+        actual = player.GetActualAccuracy();
+        base = player.GetBaseAccuracy();
+        modifier = player.uAccuracyBonus;
+    } else if (!strcmp(name, "armor_class")) {
+        actual = player.GetActualAC();
+        base = player.GetBaseAC();
+        modifier = player.sACModifier;
+    } else if (!strcmp(name, "attack_damage_melee")) {
+        actual = player.GetActualAttack(false);
+    } else if (!strcmp(name, "attack_damage_missle")) {
```

Comment:
missile?

Reply:
I have no idea about this specific binding. I ported it from the Nuklear.cpp. I didn't want to remove things which were already in place. But yes, that sounds more like a typo. I'll fix it.